clusterloader2/testing/l4ilb/config.yaml

```diff
@@ -0,0 +1,153 @@
+#Constants
+{{$LARGE_BACKEND_SIZE := 5}}
+{{$MEDIUM_BACKEND_SIZE := 3}}
+{{$SMALL_BACKEND_SIZE := 1}}
+{{$LARGE_BACKEND_LABEL := "ilblarge"}}
+{{$MEDIUM_BACKEND_LABEL := "ilbmedium"}}
+{{$SMALL_BACKEND_LABEL := "ilbsmall"}}
+# adding a fixed value for first version of the test, rate of pod creation not a concern yet.
+{{$ilbQPS := 500}}
+{{$namespaces := 1}}
+#Test
```

Comment:
nit: can you please add an empty line to make the separation clearer for readers?

Reply:
done